docs/tutorials/field_type/3_implement_the_tweet_type_class.md

```diff
@@ -11,7 +11,7 @@ A Type class must implement `eZ\Publish\SPI\FieldType\FieldType` ("Field Type in
 
 All native Field Types also extend the `eZ\Publish\Core\FieldType\FieldType` abstract class that implements this interface and provides implementation facilities through a set of abstract methods of its own. In this case, Type classes implement a mix of methods from the Field Type interface and from the abstract Field Type.
 
-The recommended way to allow the Field Type to be able to generate content name is through implementing the `eZ\Publish\SPI\FieldType\Nameable` interface.
+To allow the Field Type to retrieve content name use the `\eZ\Publish\Core\FieldType\FieldType::getName` method.
```

Comment:
```suggestion
To allow the Field Type to retrieve content name implement the `eZ\Publish\SPI\FieldType\FieldType::getName` method.
```
Please also decide it you want to precede all FQCNs with `\` or not. Sometimes it is preceded sometimes it is not. For the documentation it's just a matter of convention for readability.

Reply:
This change  was for consistency purpose exactly :)